openpype/hosts/max/api/lib.py

```diff
@@ -248,19 +248,19 @@ def get_frame_range(asset_doc=None) -> Union[Dict[str, Any], None]:
         asset_doc = get_current_project_asset()
 
     data = asset_doc["data"]
-    frame_start = data.get("frameStart")
-    frame_end = data.get("frameEnd")
+    frame_start = data.get("frameStart", 0)
+    frame_end = data.get("frameEnd", 0)
 
     if frame_start is None or frame_end is None:
         return
```

Comment:
This if statement will now never be true - I believe we don't want the fallback here - but we do want to return an empty `dict` maybe?
```suggestion
    frame_start = data.get("frameStart")
    frame_end = data.get("frameEnd")

    if frame_start is None or frame_end is None:
        return {}
```